albinos-cli/source/protocol/protocol_type.d

```diff
@@ -26,5 +26,5 @@ struct config_load_answer
 {
     @serializationKeys("REQUEST_STATE") string state;
     @serializationKeys("CONFIG_NAME") string config_name;
-    @serializationKeys("CONFIG_ID") int config_id;
```

Comment:
Why the change ?

Reply:
Config id is a size_t now

Reply:
but why would you want to switch ?

Reply:
btw, `sitez_t` != `ulong`. Such assumption is not portable :(

Reply:
@Arignir Yes. Fixed, thank's

@Gaspard-- std::hash<string>() return a size_t. That's why i switch int to size_t, config_id is now a size_t, and error are handler through a std::optional<request_state>, double check the review